src/components/panels/ConfigPanel/_config-panel.scss

```diff
@@ -8,12 +9,18 @@
     padding: 0;
   }
 
-  .is-loading {
-    &.u-vertically-center {
-      justify-content: center;
-    }
+  .full-size {
+    justify-content: center;
+    min-height: $magic-min-height;
+    text-align: center;
+  }
+
+  &__message {
+    margin-bottom: 20rem;
```

Comment:
Can't QA this yet as I can't access the model but I'm guessing this will cause issues if you reduce the height of the screen?

Reply:
I haven't noticed any qa issues.

Reply:
The empty panel scrolls off the top when on a short screen. These styles feel strange to me. Why not use Vanilla strips for padding and Vanilla alignment classes to center the content?